lms/djangoapps/courseware/access.py

```diff
@@ -173,53 +182,6 @@ def has_access(user, action, obj, course_key=None):
                     .format(type(obj)))
 
 
-# ================ Implementation helpers ================================
-
-def has_staff_access_to_preview_mode(user, obj, course_key=None):
```

Comment:
@muzaffaryousaf I have removed this method in order to implement masquerading for preview mode. Associated tests with this method has also been removed.
What are the things you may think of which can go wrong by doing this?